constants.py

```diff
@@ -1,8 +1,8 @@
 ## --- PLATFORMS --- ##
 SERVER_PLATFORMS = ["UBUNTU","CENTOS","DEBIAN","WIN","OSX","MAC", "SUSE", "OEL"]
-MOBILE_PLATFORMS = ["CBLITE", "ANDROID","IOS", "JAVA", "NET"]
+MOBILE_PLATFORMS = ["CBLITE", "ANDROID","IOS", "JAVA", "NET", "SYNCGATEWAY"]
 SDK_PLATFORMS= [".NET","JAVA","LIBC","NODE"]
-MOBILE_VERSION = "1.1"
+MOBILE_VERSION = ["1.1.0", "1.2.0"]
```

Comment:
This isn't being used now but can come in use later for selecting default mobile versions